### ISSUE TITLE
Movinets Stream Config Without PosEnc

### ISSUE THREAD

**Issue by fangwudi** [tensorflow/models#9992]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [x] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the documentation with the issue

https://github.com/tensorflow/models/blob/master/official/vision/beta/projects/movinet/configs/yaml/movinet_a2_stream_k600_8x8.yaml

## 2. Describe the issue

According to the paper, using PosEnc is good for causal model. But in stream yaml config, use_positional_encoding is not turned on. Why? @Hyperparticle

**Comment by Hyperparticle** [comment]
We are currently in the process of configuring streaming models for optimal accuracy. This includes investigating the importance of positional encoding. We have found that for the smaller models (A0-A2), the accuracy seems to be ok without positional encoding. This might lower the accuracy slightly, but it does simplify the model architecture and might make it easier to deploy on mobile devices. But for the larger models (A3-A6), we might keep the positional encoding since they are less likely to be deployed on mobile.

**Comment by fangwudi** [comment]
> We are currently in the process of configuring streaming models for optimal accuracy. This includes investigating the importance of positional encoding. We have found that for the smaller models (A0-A2), the accuracy seems to be ok without positional encoding. This might lower the accuracy slightly, but it does simplify the model architecture and might make it easier to deploy on mobile devices. But for the larger models (A3-A6), we might keep the positional encoding since they are less likely to be deployed on mobile.

Okay, got it. Thank you for your reply. When wil the demo jupyter for stream model be released? For now, it's hard to understand dataloader and training code for stream(casual) mode.

**Comment by Hyperparticle** [comment]
Yes, we will release some examples on how to use the streaming models soon, along with some pretrained checkpoints.

**Comment by Hyperparticle** [comment]
@fangwudi We have added streaming model checkpoints with examples in the README.
